exampleCourse/elementExtensions/extendable-element/extension-cssjs/info.json

```diff
@@ -3,5 +3,8 @@
     "dependencies": {
         "extensionStyles": ["extension-cssjs.css"],
         "extensionScripts": ["extension-cssjs.js"]
+    },
+    "dynamicDependencies": {
+        "nodeModulesScripts": {"d3": "d3/dist/d3.min.js"}
```

Comment:
Is this here solely for tests? Let's add this to a question in `testCourse` instead of `exampleCourse`.

Reply:
This is based on https://github.com/PrairieLearn/PrairieLearn/blob/411c2cc7a1264874b6b548fef239d66b90127ede/apps/prairielearn/src/tests/courseElementExtension.test.js#L20 which uses an example course extension for testing. I only added it to use the same example. I'm not sure why this was added there instead of testCourse, but using the testCourse, would require a bit of rewriting of that test and some reorganization of the content.

Reply:
I can understand if you don't want to do that in this PR, but it would be nice to follow up and improve this soon.

Reply:
Opened issue #8740 to keep track of this.